xonsh/ptk_shell/completer.py

```diff
@@ -98,11 +98,17 @@ def get_completions(self, document, complete_event):
         for comp in completions:
             # do not display quote
             if isinstance(comp, RichCompletion):
+                desc = (
+                    comp.description.replace(os.linesep, " ")
+                    if comp.description
+                    else None
+                )
                 yield Completion(
                     comp,
                     -comp.prefix_len if comp.prefix_len is not None else -plen,
                     display=comp.display or comp[pre:].strip("'\""),
-                    display_meta=comp.description or None,
+                    # ptk doesn't render newlines. This can be removed once it is supported.
```

Comment:
Shouldn't this comment be above the `replace` line?

Reply:
updated.